extensions/processing.adoc

```diff
@@ -2,29 +2,35 @@
 
```

Comment:
Coverages SWG call: Suggested to rename this extension to "OAPI - Coverages WCPS Extension" in order to leave room for processing of coverages for example via OAPI - Processes.